config/config.go

```diff
@@ -242,8 +242,14 @@ type BaseConfig struct { //nolint: maligned
 
 	// TCP or UNIX socket address for Ostracon to listen on for
 	// connections from an external PrivValidator process
+	// example) 0.0.0.0:26659
 	PrivValidatorListenAddr string `mapstructure:"priv_validator_laddr"`
 
+	// Validator's remote address(without port) to allow a connection
+	// ostracon only allow a connection from this address
+	// example) 10.0.0.7
+	PrivValidatorRemoteAddr string `mapstructure:"priv_validator_raddr"`
```

Comment:
I think all connections should be denied if this value is empty. It is confusing whether this value is allow list or deny list.

Reply:
I think so too. 
Actually at the first time, I implemented as you said(deny all if empty).
It works fine, but some test cases throw errors(connection timeout).
empty remote address `""` refuses connection some test cases try to establish.
I also tried to mock and set the remote address "127.0.0.1" for the precondition of tests however there seems to be more code in unit tests to fix then ip filter itself.
If this implementation is enough for now, I would implement deny all and fixes all the unit-tests when we set the policy(deny all).

Reply:
You mean, empty value is used by many tests?

Reply:
How about setting such a value that all connections are allowed (for example "*" or "all") as default value?

Reply:
Yes, empty value is used for many tests because ip filtering is new one.

For the denyAll policy(empty value), I tried to set the default value(_127.0.0.1_) in the construct for SignerListenerEndpoint for failed tests. But some of the tests instantiate SignerListenerEndpoint without calling new construct so it failed.

Since setting default value as "*", "all", "127.0.0.1" costs the same. 
I'll look at the failed tests again for denyAll.